front/app/containers/ProjectsShowPage/shared/pb/PBExpenses.tsx

```diff
@@ -407,10 +413,31 @@ class PBExpenses extends PureComponent<Props & InjectedIntlProps, State> {
             </ProgressBar>
 
             <Footer viewMode={viewMode}>
+              {viewMode === 'column' && (
+                <TotalBudgetColumn aria-hidden>
+                  <BudgetLabel>
+                    <FormattedMessage
+                      {...(minBudget === maxBudget
+                        ? messages.requiredSelection
+                        : messages.yourBudget)}
+                    />
+                    :
+                  </BudgetLabel>
+                  <BudgetAmount>
+                    <FormattedNumber
+                      value={maxBudget}
+                      style="currency"
+                      currency={currency}
+                      minimumFractionDigits={0}
+                      maximumFractionDigits={0}
+                    />
+                  </BudgetAmount>
+                </TotalBudgetColumn>
+              )}
               <Budgets>
-                <Budget aria-hidden>
+                <BudgetItem aria-hidden isLastBudgetItem={!showMinBudget}>
```

Comment:
If we don't show the min required budget, this is the last item (again, see design if you want to verify :))